Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -140,6 +161,30 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * Delete the location, triggering loading parent location and notifications
+         *
+         * @method _delete
+         * @protected
+         */
+        _delete: function () {
+            var that = this,
+                location = this.get('location'),
```

Comment:
`that` is not needed.
The `location` and `locationId` are not really needed as well, you can use the content id in the notification identifier.